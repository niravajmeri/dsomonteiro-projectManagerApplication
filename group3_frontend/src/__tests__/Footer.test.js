import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../components/footer/footer';
describe('Footer', () => {
    it('should render correctly', () => {
        const component = shallow(<Footer />);
        expect(component).toMatchSnapshot();
    });
});
