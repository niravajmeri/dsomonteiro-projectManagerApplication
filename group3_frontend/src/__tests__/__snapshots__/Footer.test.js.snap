// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="sponsor"
>
        <a
                href="https://www.isep.ipp.pt"
        >
                <img
                        alt="logo_isep"
                        className="logo_isep"
                        src="logo_isep.png"
                        width="150"
                />
        </a>
        <a
                href="https://portotechhub.com"
        >
                <img
                        alt="porto_techHub"
                        className="porto_techHub"
                        src="porto_techHub.svg"
                        width="100"
                />
        </a>
        <a
                href="http://www.portotechhub.com/switch/"
        >
                <img
                        alt="switch"
                        className="switch_icon"
                        src="switch_icon.svg"
                        width="120"
                />
        </a>
</div>,
      "className": "footer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <a
            href="https://www.isep.ipp.pt"
>
            <img
                        alt="logo_isep"
                        className="logo_isep"
                        src="logo_isep.png"
                        width="150"
            />
</a>,
          <a
            href="https://portotechhub.com"
>
            <img
                        alt="porto_techHub"
                        className="porto_techHub"
                        src="porto_techHub.svg"
                        width="100"
            />
</a>,
          <a
            href="http://www.portotechhub.com/switch/"
>
            <img
                        alt="switch"
                        className="switch_icon"
                        src="switch_icon.svg"
                        width="120"
            />
</a>,
        ],
        "className": "sponsor",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="logo_isep"
              className="logo_isep"
              src="logo_isep.png"
              width="150"
/>,
            "href": "https://www.isep.ipp.pt",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "logo_isep",
              "className": "logo_isep",
              "src": "logo_isep.png",
              "width": "150",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="porto_techHub"
              className="porto_techHub"
              src="porto_techHub.svg"
              width="100"
/>,
            "href": "https://portotechhub.com",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "porto_techHub",
              "className": "porto_techHub",
              "src": "porto_techHub.svg",
              "width": "100",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="switch"
              className="switch_icon"
              src="switch_icon.svg"
              width="120"
/>,
            "href": "http://www.portotechhub.com/switch/",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "switch",
              "className": "switch_icon",
              "src": "switch_icon.svg",
              "width": "120",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "a",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="sponsor"
>
          <a
                    href="https://www.isep.ipp.pt"
          >
                    <img
                              alt="logo_isep"
                              className="logo_isep"
                              src="logo_isep.png"
                              width="150"
                    />
          </a>
          <a
                    href="https://portotechhub.com"
          >
                    <img
                              alt="porto_techHub"
                              className="porto_techHub"
                              src="porto_techHub.svg"
                              width="100"
                    />
          </a>
          <a
                    href="http://www.portotechhub.com/switch/"
          >
                    <img
                              alt="switch"
                              className="switch_icon"
                              src="switch_icon.svg"
                              width="120"
                    />
          </a>
</div>,
        "className": "footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <a
              href="https://www.isep.ipp.pt"
>
              <img
                            alt="logo_isep"
                            className="logo_isep"
                            src="logo_isep.png"
                            width="150"
              />
</a>,
            <a
              href="https://portotechhub.com"
>
              <img
                            alt="porto_techHub"
                            className="porto_techHub"
                            src="porto_techHub.svg"
                            width="100"
              />
</a>,
            <a
              href="http://www.portotechhub.com/switch/"
>
              <img
                            alt="switch"
                            className="switch_icon"
                            src="switch_icon.svg"
                            width="120"
              />
</a>,
          ],
          "className": "sponsor",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="logo_isep"
                className="logo_isep"
                src="logo_isep.png"
                width="150"
/>,
              "href": "https://www.isep.ipp.pt",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "logo_isep",
                "className": "logo_isep",
                "src": "logo_isep.png",
                "width": "150",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "a",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="porto_techHub"
                className="porto_techHub"
                src="porto_techHub.svg"
                width="100"
/>,
              "href": "https://portotechhub.com",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "porto_techHub",
                "className": "porto_techHub",
                "src": "porto_techHub.svg",
                "width": "100",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "a",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="switch"
                className="switch_icon"
                src="switch_icon.svg"
                width="120"
/>,
              "href": "http://www.portotechhub.com/switch/",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "switch",
                "className": "switch_icon",
                "src": "switch_icon.svg",
                "width": "120",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "a",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
